frontend/simple/components/CreateGroup/CreateGroupSummary.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div>
+    <h1 id="summaryStep" class="title is-1 has-text-centered"><i18n>Review & Finish</i18n></h1>
+
+    <h2 class="title is-6"><i18n>Group name:</i18n></h2>
+    <p class="subtitle is-2" :class="!group.groupName && 'is-danger'">{{ group.groupName || 'No group name set' }}</p>
```

Comment:
1. This should probably use vee-validate or something else?
2. If it's not in an `<i18n>` tag, it must be in an `L` function.

(Same goes for all other such instances, e.g. below on line 18 and if necessary in other files)